Userland/Libraries/LibCore/System.cpp

```diff
@@ -1055,6 +1055,28 @@ ErrorOr<void> utime(StringView path, Optional<struct utimbuf> maybe_buf)
 #endif
 }
 
+ErrorOr<void> utimensat(int fd, StringView path, struct timespec const times[2], int flag)
+{
+    if (path.is_null())
+        return Error::from_errno(EFAULT);
+
+#ifdef AK_OS_SERENITY
+    Syscall::SC_utimensat_params params {
```

Comment:
Do we not need any of the checks that are listed here, at least for the SerenityOS case?

https://github.com/SerenityOS/serenity/blob/82219d0879aee5d47abccfeab6b7ca5894e5e522/Userland/Libraries/LibC/fcntl.cpp#L137-L162